spring-boot-autoconfigure/src/main/java/org/axonframework/springboot/autoconfig/AxonServerAutoConfiguration.java

```diff
@@ -207,6 +135,7 @@ public void setApplicationContext(ApplicationContext applicationContext) throws
     }
 
     @Bean
+    @ConditionalOnMissingBean
     public EventProcessorInfoConfiguration processorInfoConfiguration(
```

Comment:
I thought we had replaced the creation of the `EventProcessorInfoConfiguration` for the `EventProcessorControlService`?